timings/accuracy/utils.py

```
@@ -0,0 +1,748 @@
+
+    Args:
+        f_l: Lower frequency bound (Hz).
+        f_u: Upper frequency bound (Hz).
+        f_sampling: Sampling frequency (Hz).
+        T: Duration (seconds).
+
+    Returns:
+        Frequency array in the range (f_l, f_u).
+    """
+    delta_t = 1 / f_sampling
+    tlen = int(round(T / delta_t))
```

Ruff (0.16.0)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)